apps/src/codebridge/FileBrowser/FileBrowserHeaderPopUpButton.tsx

```diff
@@ -13,19 +13,12 @@ import {
   useHandleFileUpload,
   usePrompts,
 } from './hooks';
-import {newFilePromptType} from './types';
```

Comment:
`FileBrowserHeaderPopUpButton` now no longer needs any args from the `FileBrowser` and just snags both functions from the `usePrompts` hook.